src/Ext/freecad/CMakeLists.txt

```diff
@@ -10,7 +10,11 @@ if (WIN32)
                        REALPATH BASE_DIR "${CMAKE_INSTALL_PREFIX}")
```

Comment:
actually looking at https://cmake.org/cmake/help/latest/command/get_filename_component.html it seems the windows portion is already doing what I want, what's the reason for having this only for windows?

Reply:
ok, it's not the same actually since it's using CMAKE_INSTALL_BINDIR, can someone explain why BINDIR is used for windows?

Reply:
sorry I can't remember.